### PR TITLE
patch: added missing py runtime updates

### PR DESCRIPTION
**Pull request by BagchiMB**
### What does this PR do?
Updates py runtime.

### Motivation
I was working on the setup while working as my lambda was running at python 3.10 and on the documentation I didn't see python 3.10 mentioned, for a wild guess I checked with `Python310` and verified the arn it worked, therefore I also checked with `Python311` it also got verified, therefore just a small PR.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
@BagchiMB Thanks for submitting this PR! I've created DOCS-5995 to track this update. Someone on the Docs Team will be in touch with you after we've validated it with the appropriate engineering team. Thanks!

**Comment by BagchiMB** [comment]
@drichards-87 any update on this?

**Comment by kayayarai** [comment]
Confirmed by eng. Thanks for the PR, @BagchiMB !

File: content/en/security/application_security/enabling/serverless.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ To install and configure the Datadog Serverless Framework plugin:
           # Use this format for arm64-based Lambda deployed in AWS GovCloud regions
           arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:Datadog-<RUNTIME>-ARM:72
           ```
-          Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38` and `Python39`.
+          Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38`, `Python39`, `Python310` and `Python311`.
```
Comment (kayayarai) [inline content/en/security/application_security/enabling/serverless.md:158]:
```suggestion
          Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38`, `Python39`, `Python310`, and `Python311`.
```

File: content/en/serverless/installation/python.md [hunk 2]
```diff
@@ -272,7 +272,7 @@ The [Datadog CDK Construct][1] automatically installs Datadog on your functions
       arn:aws-us-gov:lambda:<AWS_REGION>:002406178527:layer:Datadog-<RUNTIME>-ARM:{{< latest-lambda-layer-version layer="python" >}}
       ```
 
-      Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38` and `Python39`.
+      Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38`, `Python39`, `Python310` and `Python311`.
```
Comment (kayayarai) [inline content/en/serverless/installation/python.md:275]:
```suggestion
      Replace `<AWS_REGION>` with a valid AWS region, such as `us-east-1`. The available `RUNTIME` options are `Python37`, `Python38`, `Python39`, `Python310`, and `Python311`.
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**